backend/internal/handlers/webhooks/webhooks.go

```diff
@@ -166,7 +155,12 @@ func (s *WebHookService) updateWorkStateOnStudentCommit(c *fiber.Ctx, pushEvent
 	return c.SendStatus(200)
 }
 
-func (s *WebHookService) isInitialCommit(pushEvent github.PushEvent) (bool, error) {
-	isInitialCommit := pushEvent.BaseRef == nil && *pushEvent.Created && pushEvent.GetBefore() == "0000000000000000000000000000000000000000"
-	return isInitialCommit, nil
+func (s *WebHookService) isInitialCommit(pushEvent github.PushEvent) bool {
+	return pushEvent.BaseRef == nil && *pushEvent.Created && pushEvent.GetBefore() == "0000000000000000000000000000000000000000"
+}
+
+func (s *WebHookService) isBotPushEvent(pushEvent github.PushEvent) bool {
+	return pushEvent.Pusher != nil &&
+		pushEvent.Pusher.Name != nil &&
+		strings.Contains(*pushEvent.Pusher.Name, "[bot]")
 }
```

Comment:
i thought we talked about this before because i did that and found that without the (s *WebHookService) its a global function? correct me if i misremember
in any case, both this and the isInitialCommit function should be in the same scope so lmk

Reply:
They'd become a global function if they were capitalized but here they are private because they're lowercase.